cmd/snap-confine/mount-support-nvidia.c

```diff
@@ -81,19 +81,10 @@ static const size_t egl_vendor_globs_len =
 // FIXME: this doesn't yet work with libGLX and libglvnd redirector
```

Comment:
this comment should at least be modified to explain why we are excluding certain directories, it's unclear to me if the TODO here is fully resolved though

Reply:
This block of comments appears to be unchanged since 2016-06-08. We've clearly been using and relying on glvnd in snaps since core18 (both with and without Nvidia drivers), so I think this has been out of date for a long time.

I'm honestly not sure how much of the comment should be kept. The archlinux links all seem to give 404 errors now too.

Reply:
It's probably reasonable to just delete the whole comment then and leave a comment about the current state of affairs, mainly that we don't mount certain libraries because they have incompatible glibc dependencies, etc